### ISSUE TITLE
Translate ON01

### ISSUE THREAD

**Issue by edwardsmarc** [edwardsmarc/CASFRI#558]
Appears to use an older version of the FIM standard.

**Comment by MelinaHoule** [comment]
Here are some notes from 2015: Major bug in conversion of Disturbance in Ontario. Disturbance is derived from DEVSTAGE column. DEVSTAGE has an entry for every rows and each value has a translation rule. Which means every forested polygons in ON is considered as being disturbed or resulting from a disturbance. Disturbance year is 80% of the time unknown. For example: FTGNAT (Productive forest areas which were regenerated predominantly by natural means) is translated as BURN according to Cosco. It shouldn't. Some DEVSTAGE are specifically related to sylviculture treatment. We should split the real known disturbance than what could be the origin of the stand. 

Conversion rules need to be revisited. Many assumptions are completely wrong according to supported documentation.

**Comment by edwardsmarc** [comment]
Flag for @prvernier : check that the DST rules are the same for ON01 and ON02. If not we will need a custom function to run the correct mappings based on incentory_id.

**Comment by prvernier** [comment]
There are 4 types of issues with DST, specifically with DIST_TYPE_1:

1. ON01 contains abbreviations of classes that ON02 doesn't use. This one is resolved by adding each abbreviated version of the classes to the translation table.
2. ON01 contains several classes that are not mentioned in either the Cosco specs or the Ontario docs: LOWS, SCAR, SIPM, TENA, TENG. Currently, these are considered to be errors.
3. ON02 contains one class that is not mentioned in either the Cosco specs or the Ontario docs: REGENASS. Currently, this is considered to be an error.
4. In CAS04, the classes LOWNAT, DEPNAT, and FTGNAT are coded as BU (burns). Currently, in CAS05, these are considered errors since they are naturally regenerating stands (see above discussion). There is an additional class, NEWNAT, which is not mentioned in Cosco but which we also consider as an error (for now).

Should we contact Ontario folks about point 2 & 3?

Should we revisit point 4?

**Comment by edwardsmarc** [comment]
Yes maybe @MelinaHoule has a contact in ON? Or you could check the contact in the inventory_list.

**Comment by prvernier** [comment]
Our contact in Ontario is Larry Watkins but he hasn't responded to two queries sent by Melina about species codes. I found a different contact name (Ken Smith; ken.smith@ontario.ca) whom I've just emailed about DEVSTAGE values. If he responds, I'll update the contact list and maybe Melina could also email him with her original species queries.

**Comment by edwardsmarc** [comment]
For now we have removed the codes mentioned in point 4 above. We are waiting for more info on points 2 and 3.
